packages/amplify-util-uibuilder/commands/utils/syncAmplifyUiBuilderComponents.js

```diff
@@ -30,19 +31,15 @@ async function listUiBuilderComponents(context, envName) {
         environmentName,
       })
       .promise();
-    logger.info(JSON.stringify(uiBuilderComponents, null, 2));
+    printer.debug(JSON.stringify(uiBuilderComponents, null, 2));
     return uiBuilderComponents;
   } catch (e) {
-    logger.error(e);
+    printer.debug(e);
```

Comment:
printer.error? (also applies to the rest of this file)

Reply:
Do we want to show rich error stack traces without the --debug flag? Or just the `X components failed to load` notice from the ora spinner?